Service/cron.job.js

```diff
@@ -16,6 +17,15 @@ const experimentStatusUpdate = cron.schedule("* 0 * * *", async () => {
     });
 })
 
+cron.schedule("0 0 1 * *", async () => {
+  try {
+    await Experiment.updateMany({}, { $set: { monthlyCallCount: 0 } });
+    console.log("Monthly call count reset successful");
```

Comment:
although you are correct that we should log this data, console log is no the way of doing this